tests/test_point_cloud_scene.py

```diff
@@ -0,0 +1,35 @@
+import wandb
+import numpy as np
+
+wandb.init()
+
+wandb.log(
+    {
+        "point_scene": wandb.Object3D(
+            {
+                "type": "scene/v1",
+                "points": np.array([[0.4, 1, 1.3], [1, 1, 1], [1.2, 1, 1.2]]),
+                "boxes": np.array(
+                    [
+                        {
+                            "x": 0,
+                            "y": 0,
+                            "height": 3,
+                            "width": 2,
+                            "depth": 1,
+                        },
+                        {
+                            "x": 0.3,
+                            "y": 0,
+                            "height": 0.2,
+                            "width": 0.2,
+                            "depth": 4,
+                        }
+                    ]
+                ),
+                "center": [1, 1, 1],
+            }
```

Comment:
Is this generated by hand in the real world?  Will the arrays always be numpy?

Reply:
@vanpelt In case of the lyft dataset they're stored as (**id, string**) combos, where the string is in the format: 
**center_x, center_y, center_z, width, length, height, yaw, class_name**

I split them and convert them into numpy arrays.

Reply:
@vanpelt They'll most likely be in a couple different formats, tensor, numpy, or a list. I keep the points as numpy to keep the API consistent, because that's the format we accept for the other media when it's raw data.

Reply:
I think we need a unified way of representing this structure.  FastaiV2 has typed tensors which we should copy...  we need a unified pythonic way to create these data structures for this to be usable.

Reply:
> we need a unified pythonic way to create these data structures for this to be usable.

I don't understand what you're asking for.  Adding a new data type is Are we going to write another python list library? Do we want to accept a polymoprhic batch of python list types, e.g. pytorch tensor, tf tensor, numpy array, list, etc...

What's the problem we're solving here?

Reply:
The problem I'm trying to solve is expecting users to create these arbitrary structures by hand without screwing things up.  If this custom data structure really is the only way to support bounding boxes then we at least need a schema and validation.

Reply:
And regarding the handling of lists, we already have logic to convert all list like things to actual python lists.  I would expect users could use any of the list types you mentioned to populate this structure.